dnf.spec

```diff
@@ -232,6 +239,9 @@ ln -sr  %{buildroot}%{confdir}/protected.d %{buildroot}%{_sysconfdir}/yum/protec
 ln -sr  %{buildroot}%{confdir}/vars %{buildroot}%{_sysconfdir}/yum/vars
 %endif
 
+%if 0%{?fedora} > 38
+rm %{buildroot}%{confdir}/%{name}.conf
```

Comment:
So here when installing both new versions of dnf and dnf5 we want probably to have the dnf one go first to remove the file and then dnf5 one will install new one. But can we assure this order?

Reply:
the file is un-owned by dnf so it would not be touched, the only time dnf touches the file is at buildtime. dnf5 will install the file instead

Reply:
I am sorry, now I realized it's a removal from the buildroot and not like from the actually installed files...